includes/managers/class-fs-admin-menu-manager.php

```diff
@@ -360,9 +360,16 @@ function get_top_level_menu_slug() {
 		 * @return bool
 		 */
 		function is_activation_page() {
-			return isset( $_GET['page'] ) &&
-			       ( ( strtolower( $this->_menu_slug ) === strtolower( $_GET['page'] ) ) ||
-			         ( strtolower( $this->_plugin_slug ) === strtolower( $_GET['page'] ) ) );
+			$fs = freemius( $this->_plugin_slug );
+
+			if ( $fs->get_plugin()->is_plugin() ) {
+				return isset( $_GET['page'] ) &&
+				       ( ( strtolower( $this->_menu_slug ) === strtolower( $_GET['page'] ) ) ||
```

Comment:
@fajardoleo there's a helper function of that (and the following line: `fs_is_plugin_page()`